api/tests/test_add_response_headers.py

```diff
@@ -0,0 +1,25 @@
+from api.security.headers import SECURITY_HEADERS, CORS_HEADERS
+
+
+def test_response_security_headers_dev(dev_app, dev_client):
+    """
+    Given a PCM app in local development
+      When a request is performed
+        Then it should have security headers correctly set for local development
+    """
+    response = dev_client.get('/manager/get_app_config')
```

Comment:
I love the idea of testing it by hitting the actual endpoint, however this only test one endpoint. I understand it implicitly tests the app extension as well, but I'd be more explicit and directly test either the `SecurityHeaders` class or the two functions (`add_security_headers_`) to make sure they add the full list of header to a given Response object

It's a bit of a small detail, I'll leave it up to you to decide whether to change this or not

Reply:
Done in this latest review, I believe your suggestion is a great value so I implemented it
Thanks!